packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx

```diff
@@ -129,6 +132,14 @@ const getDocs: () => JSX.Element = () => {
         button will cycle through the different camera's in the users device. Typical usage is to enable this button on
         mobile devices. This button is enabled through the use of the `showCameraSwitcherInLocalPreview` prop.
       </Description>
+      <StorybookBanner
+        // eslint-disable-next-line react/no-children-prop
```

Comment:
Why do you need this? I don't see any explicit `children` prop here.

Reply:
oops I think I forgot to remove that!

Reply:
fixed!